tests/terraform/module_loading/data/tf_managed_modules/main.tf

```diff
@@ -12,3 +12,21 @@ module "log_group_v4" {
   name_prefix       = "my-log-group-"
   retention_in_days = 7
 }
+
+# Need to verify this type of comment is not an issue.
```

Comment:
it's not clear from the test that this is a part of the test and not just commented out code.
I suggest using CAPS to clarify, something like:
`NOTE - THIS IS COMMENTED AS PART OF THE TEST TO CHECK HANDLING OF COMMENTED OUT MODULES. DO NOT DELETE`

Reply:
i will add this, thanks!